go-server/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Используем официальный образ Go для сборки приложения
+FROM golang:1.16-alpine AS builder
+
+# Устанавливаем рабочую директорию внутри контейнера
+WORKDIR /app
+
+# Копируем go.mod и go.sum и устанавливаем зависимости
+COPY go.mod go.sum ./
```

Comment:
what is the reason for this copy?

Reply:
It's hard to explain, it was swearing at something, that's why I added it.

Reply:
Remake build.
It should be much more simple like
```
COPY . . 
RUN go build -o app
```